source/agora/common/crypto/Key.d

```diff
@@ -102,14 +114,11 @@ unittest
 /// Represent a public key / address
 public struct PublicKey
 {
-    /// Alias to the BitBlob type
-    private alias DataType = BitBlob!(crypto_sign_ed25519_PUBLICKEYBYTES * 8);
-
-    /*private*/ DataType data;
+    /*private*/ Point data;
```

Comment:
The commit message is wrong. 
Public key is Point and Secret key is Scalar.

Reply:
Updated. Ty.